std/file.d

```diff
@@ -2983,6 +2983,31 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
     return DirIterator(path, mode, followSymlink);
 }
 
+/// Duplicate functionality of D1's $(D std.file.listdir()):
+unittest
+{
+  string[] listdir(string pathname)
```

Comment:
Agreed with @quickfur. In particular:

> I redid all the examples in the language spec to be tighter and work on the Kindle, but they've been bit rotting since and things have been reverting to being unreadable on the Kindle.

Shows that this is not a good solution. I also don't believe in having different indentation in doc/non-doc, since you can take any valid piece of code, slap on `///`, and it becomes a documented unittest.

I wouldn't be against changing the *entire* style guide to using 2 space indent though....

...or just tabs (gasp!)